### PR TITLE
systemd: Don't force enable units

### PR DESCRIPTION
**Pull request by brunnre8**
Fixes: https://github.com/thelounge/thelounge-deb/issues/70

### CODE REVIEW

File: build-package [hunk 1]
```diff
@@ -37,12 +37,9 @@ install -Dm660 \
 
 # Install systemd units
 install -Dm644 "$STARTDIR/systemd/system.service" \
-	"$DESTDIR/lib/systemd/system/thelounge.service"
+	"$DESTDIR/usr/lib/systemd/system/thelounge.service"
```
Comment (maxpoulin64) [inline build-package:40]: This is incorrect: on Debian/Ubuntu, the folder is indeed `/lib/systemd/system` contrary to Arch. This fails to install in `/usr/lib/systemd/system`.

**Review by maxpoulin64 — CHANGES_REQUESTED** [review]

File: build-package (hunk 1 above)
Comment (maxpoulin64) [reply build-package:40]: Also note that this is only for system. For user units it's indeed `/usr/lib` because Debian

Comment (brunnre8) [reply build-package:40]: You gotta be kidding me... Thanks!
Sorry I'm not used to debianisms.

**Comment by brunnre8** [comment]
fixed it up

**Pull request merged**